Junction/Junction/JunctionKeeper.swift

```diff
@@ -87,4 +89,30 @@ internal final class JunctionKeeper {
         
         return previousValues!.writeToFile(path, atomically: true)
     }
+    
+    internal func deleteValueFromArray(key: String, valueToDelete: AnyObject) -> Bool {
+        createPlistIfNeeded()
+        
+        let filename = "JunctionData.plist"
+        let documentDirectory = NSSearchPathForDirectoriesInDomains(.DocumentDirectory, .UserDomainMask, true)[0] as String
+        let path = documentDirectory.stringByAppendingString("/\(filename)")
```

Comment:
You should probably have all the code that gets the path to the plist in one place. Guessing these same 3 lines are in a few places throughout the code base.